lib/matplotlib/contour.py

```diff
@@ -92,14 +90,16 @@ def clabel(self, levels=None, *,
             This spacing will be exact for labels at locations where the
             contour is straight, less so for labels on curved contours.
 
-        fmt : str or dict, default: '%1.3f'
-            A format string for the label.
+        fmt : `.Formatter` or str or callable or dict, optional
+            How the levels are formatted:
 
-            Alternatively, this can be a dictionary matching contour levels
-            with arbitrary strings to use for each contour level (i.e.,
-            fmt[level]=string), or it can be any callable, such as a
-            `.Formatter` instance, that returns a string when called with a
-            numeric contour level.
+            - If a `.Formatter`, it is used to format all levels at once, using
+              its `.Formatter.format_ticks` method.  The default is to use a
+              standard `.ScalarFormatter`.
```

Comment:
I would put this default at the end outside of the list; it is not a default specific to this list item.

Reply:
yes